static/js/services/delete-doc.js

```diff
@@ -8,6 +8,9 @@
     if (doc.type === 'person' && doc.parent && doc.parent._id) {
       db.getDoc(doc.parent._id, function(err, parent)  {
         if (err) {
+          if (err.reason === "deleted") {
+            return callback();
+          }
```

Comment:
Know what you mean.  But until these database updates are (a) happening and (b) atomic, then this kind of issue is possible.  I spoke briefly with @browndav about this - he may have some useful input.